google/cloud/logging_v2/client.py

```diff
@@ -376,7 +376,7 @@ def get_default_handler(self, **kw):
             if monitored_resource.type == _GAE_RESOURCE_TYPE:
                 return CloudLoggingHandler(self, resource=monitored_resource, **kw)
             elif monitored_resource.type == _GKE_RESOURCE_TYPE:
-                return ContainerEngineHandler(**kw)
+                return StructuredLogHandler(**kw, project_id=self.project)
```

Comment:
nit: I think that I saw this commit in PR #450. Is it possible there had to be a rebase?

Reply:
Yeah, it seems like there was a merging issue in the v3.0.0 branch at some point which changed this line, and I had to fix it in both branches for tests to pass